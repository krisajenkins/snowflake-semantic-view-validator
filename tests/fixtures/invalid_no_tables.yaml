name: NO_TABLES
description: This semantic model has no tables
tables: []
