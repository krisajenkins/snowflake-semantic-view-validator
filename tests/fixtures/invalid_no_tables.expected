════════════════════════════════════════════════════════════════════════════════
  VALIDATION ERROR
════════════════════════════════════════════════════════════════════════════════

✗ Semantic model must have at least one table

════════════════════════════════════════════════════════════════════════════════